funnel/templates/proposal.html.jinja2

```diff
@@ -13,233 +12,201 @@
-      <div class="slider-btn {% if proposal.getnext() %} slider-btn--push-down {% endif %} mui--pull-left">
-        {%- if proposal.getprev() -%}
-          <p class="mui--text-title mui--text-light {%- if proposal.getnext() %} mui--hidden-xs mui--hidden-sm{%- endif -%}">Previous proposal</p>
-          <a href="{{ proposal.url_for('prev') }}" rel="nofollow" class="mui--text-title mui--text-light mui--text-bold slider-btn__action-btn">{{ faicon(icon='arrow-left', icon_size='subhead', baseline=false) }} <span class="truncate-title {%- if proposal.getnext() %} mui--hidden-xs mui--hidden-sm{%- endif -%}">&nbsp;{{ proposal.getprev().title }}</span></a>
-        {%- endif -%}
+{% macro share_widget() %}
+  <a href="javascript:void(0)" class="hg-link-btn mui--hide mui--text-title mui-text-light" data-action="Share dropdown" data-cy="share-project" data-url="{{ url }}&amp;utm_campaign=webshare">{{ faicon(icon='share-alt', icon_size='title', baseline=false, css_class='mui--text-light fa-icon--margin ') }}</a>
+  <div class="mui-dropdown">
+    <a href="javascript:void(0)" class="project-links mui--text-title mui-text-light" data-mui-toggle="dropdown" data-action="Share dropdown" data-cy="share-project">{{ faicon(icon='share-alt-square', icon_size='title', baseline=false, css_class='mui--text-light fa-icon--margin ') }}</a>
+    {{ share_dropdown(url, title) }}
+  </div>
+  {% if 'edit_proposal' in proposal.permissions(current_auth.user) or proposal.current_roles.project_editor %}
+    <a class="mui--text-title mui--text-light mui--d-inlineblock" href="{{ proposal.url_for('admin') }}" rel="modal:open" aria-label="{% trans %}Admin panel{% endtrans %}" data-cy="admin-panel">{{ faicon(icon='cog', icon_size='title', baseline=false) }}</a>
+  {% endif %}
+{% endmacro %}
+
+{% block contenthead %}
+  <div class="content-head content-head--top-padding mui--bg-primary mui--hidden-xs mui--hidden-sm">
+    <div class="mui-container">
+      <div class="grid">
+        <div class="grid__col-xs-12">
+          {{ project_mini_header(project) }}
+        </div>
       </div>
-      <div class="slider-btn slider-btn--right mui--pull-right">
-        {%- if proposal.getnext() -%}
-          <p class="mui--text-title mui--text-light mui--text-right">Next proposal</p>
-          <a href="{{ proposal.url_for('next') }}" rel="nofollow" class="mui--text-title mui--text-light mui--text-bold mui--text-right slider-btn__action-btn"><span class="truncate-title">{{ proposal.getnext().title }}</span> {{ faicon(icon='arrow-right', icon_size='subhead', baseline=false) }}</a>
-        {%- endif -%}
+    </div>
+  </div>
+  <div class="mui--hidden-md mui--hidden-lg mui--hidden-xl">
+    <div class="mobile-nav mui--z1">
+      <a href="{{ proposal.project.url_for() }}" aria-label="{% trans %}Back to the project{% endtrans %}" class="mui--text-dark mobile-nav__icon" data-action="Back to the {{ project.title }}(proposal page)">{{ faicon(icon='arrow-left', icon_size='title') }}</a>
```

Comment:
Where are these `data-action` attributes used? Unclear if they need translation or not.